src/rules.jl

```diff
@@ -217,8 +222,8 @@ _abs_deriv(x) = signbit(x) ? -one(x) : one(x)
 @define_diffrule NaNMath.acosh(x)  = :(  inv(NaNMath.sqrt(NaNMath.pow($x, 2) - 1))  )
 @define_diffrule NaNMath.atanh(x)  = :(  inv(1 - NaNMath.pow($x, 2))                )
 @define_diffrule NaNMath.log(x)    = :(  inv($x)                                    )
-@define_diffrule NaNMath.log2(x)   = :(  inv($x) / NaNMath.log(2)                   )
-@define_diffrule NaNMath.log10(x)  = :(  inv($x) / NaNMath.log(10)                  )
+@define_diffrule NaNMath.log2(x)   = :(  inv($(DiffRules.logtwo) * $x)  )
+@define_diffrule NaNMath.log10(x)  = :(  inv($(DiffRules.logten) * $x)  )
```

Comment:
@devmotion I remembered why I did this: https://github.com/mlubin/NaNMath.jl/issues/47